pykumo/py_kumo.py

```diff
@@ -262,6 +263,32 @@ def get_defrost(self):
             val = None
         return val
 
+    def get_hold_time(self):
+        """ Get hold time from MHK2 """
+        query = '{"c":{"mhk2":{"hold":{"adapter":{"endTime":{}}}}}}'.encode('utf-8')
+        response = self._request(query)
+        try:
+            end_time = response['r']['mhk2']['hold']['adapter']['endTime']
+        except KeyError:
+            end_time = None
```

Comment:
end_time = None is going to trigger a TypeError down at line 284.  Maybe defaulting to 0 is a better choice?

Reply:
@cjkrolak Thanks! Pushed a change where I log in this case and set to "".